news/_posts/releases/2020-04-20-release-note-1765.md

```diff
@@ -0,0 +1,102 @@
+
+Back-office regressions:
+
+- When adding a cart rule to an order from the back-office, the value discount was not correct [#18630](https://github.com/PrestaShop/PrestaShop/issues/18630)
+- Searching a category with the quick search no longer redirected to the category edition page - [#17908](https://github.com/PrestaShop/PrestaShop/issues/17908)
+- The help card was no longer displayed on view order and new employee pages - [#18279](https://github.com/PrestaShop/PrestaShop/issues/18279) and [#18615](https://github.com/PrestaShop/PrestaShop/issues/18615)
+- In the customer view page, the number of “last emails” was incorrect - [#18602](https://github.com/PrestaShop/PrestaShop/issues/18602)
+- It was not possible to access the translation interface for the Serbian language - [#18062](https://github.com/PrestaShop/PrestaShop/issues/18062)
+
+
+## Security fixes
+
+Some security fixes have been included in this patch version to ensure an improved core reliability. Thanks a lot to Rabhi for finding a lot of these issues !
+
+Improper access controls:
+
+- on product page with combinations, attachments and specific prices ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-cvjj-grfv-f56w))
+- on product attributes page ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-4wxg-33h3-3w5r))
+- on customers search ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-r6rp-6gv6-r9hq))
+- on several other pages ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-74vp-ww64-w2gm))
+
+Reflected XSS:
+
+- related in import page ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-98j8-hvjv-x47j))
+- with back parameter ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-j3r6-33hf-m8wh))
+- on Exception page ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-mrpj-67mq-3fr5))
+- on AdminCarts page ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-q6pr-42v5-v97q))
+- on Search page ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-rpg3-f23r-jmqv))
+- on dashboard calendar ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-m2x6-c2c6-pjrx))
+- on AdminFeatures page ([security advisor](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-87jh-7xpg-6v93))
+- on AdminAttributesGroups page ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-7fmr-5vcc-329j))
+- in security compromised page ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-48vj-vvr6-jj4f))
+- when using back parameter ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-375w-q56h-h7qc))
```

Comment:
what's the difference with the first element "with back parameter" ?

Reply:
@PierreRambaud any info?

Reply:
Because the first is a reflected XSS, this one is an open redirection, it's not the same issue.
```suggestion

Open redirection:

- when using back parameter ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-375w-q56h-h7qc))
```